src/banner/banner.component.ts

```diff
@@ -38,7 +40,7 @@ import { NotificationContent } from "./banner-content.interface";
 			type="button">
 			<svg
 				class="bx--inline-notification__close-icon"
-				aria-label="close"
+				[attr.aria-label]="bannerObj.closeLabel"
```

Comment:
this label should be on the button to fix DAP violation: `Form controls must have associated labels`. See #194 for more clarification

Reply:
if it's in that PR I don't need to fix it here as well.